### PR TITLE
create .gitignore

### LINKED ISSUE #5: edit .gitignore
.gitignore file needs to modify to exclude some more files.

### PR DESCRIPTION
**Pull request by jwpark-sungshin**
exclude *.o, *.out from git

resolved: #5

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.o
```
Comment (pjbear1215) [inline .gitignore:1]:
```suggestion
*.out
*.o
*.exe
```

**Review by pjbear1215 — CHANGES_REQUESTED** [review]
This is looking greate. I've left a few comments that should be addressed before this gets merged.

File: .gitignore (hunk 1 above)
Comment (jwpark-sungshin) [reply .gitignore:1]: OK. I'll do it.

**Review by pjbear1215 — APPROVED** [review]
Well done!

**Pull request merged**